internal/cmdsync/delete/command.go

```diff
@@ -182,7 +178,15 @@ func (r *runner) runE(cmd *cobra.Command, args []string) error {
 	}
 
 	if err := r.client.Delete(r.ctx, &rs); err != nil {
-		return errors.E(op, err)
+		return errors.E(op, fmt.Errorf("failed to clean up RootSync: %w", err))
```

Comment:
I wonder if we should continue deleting when we encounter (some) errors and report aggregate errors at the end.

Reply:
In this case there isn't much more we can do if we can't delete this one. CS will recreate the RG if we delete it.